metrics/examples/grafana/strimzi-kafka.json

```diff
@@ -1072,6 +1072,342 @@
         "alignLevel": null
       }
     },
+    {
+      "cacheTimeout": null,
+      "colorBackground": false,
+      "colorValue": true,
+      "colors": [
+        "#d44a3a",
+        "rgba(237, 129, 40, 0.89)",
+        "#299c46"
+      ],
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "Total Incoming Byte Rate",
```

Comment:
Are we consistently using "Rate" to mean "Per Second" across all the dashboard (I have no idea)?

Reply:
We are, just the messages in per second is different because it's the exact name of the metric got from Kafka while the others come from totals on which the irate prometheus function is applied. I agree on using Rate everywhere.